generated/ruby/test/test_errorreporting.rb

```diff
@@ -0,0 +1,101 @@
+#
```

Comment:
The file name itself -- python seems to have `generated/python/test/vision.py`, why does this have `test_` prefix?